website/notifications/utils.py

```diff
@@ -175,7 +175,7 @@ def get_configured_projects(user):
         # If the user has opted out of emails skip
         node = subscription.owner
 
-        if not isinstance(node, Node) or (user in subscription.none and not node.parent_id):
+        if not isinstance(node, Node) or (user in subscription.none and not node.parent_id) or node.title == 'Bookmarks':
```

Comment:
You can check `node.is_bookmark_collection` rather than comparing against the title.